scripts/variantstore/wdl/GvsCalculatePrecisionAndSensitivity.wdl

```diff
@@ -297,6 +294,35 @@ task Add_AS_MAX_VQS_FIELD_ToVcf {
   }
 }
 
+task IsVQSRLite {
+  input {
+    File input_vcf
+  }
+
+  String is_vqsr_lite_file = "is_vqsr_lite_file.txt"
+
+  command {
+    set +e
+
+    grep -v '^#' ~{input_vcf} | grep AS_VQS_SENS > /dev/null
```

Comment:
I know what this is all doing, but I think a lil comment would be really helpful for when I (and others) forget in a few weeks

Reply:
Okay.